service/history/execution/context.go

```diff
@@ -1182,11 +1197,8 @@ func (c *contextImpl) createWorkflowExecutionWithRetry(
 		// workflow ID reuse policy
 		return nil, err
 	default:
-		c.logger.Error(
+		logger.Error(
 			"Persistent store operation failure",
-			tag.WorkflowID(c.workflowExecution.GetWorkflowID()),
-			tag.WorkflowRunID(c.workflowExecution.GetRunID()),
-			tag.WorkflowDomainID(c.domainID),
```

Comment:
why removing these log tags? It's fine if this function needs to live detached from `contextImpl` but let's not lose these useful tags. Pass them as parameter maybe?

Reply:
it's not removed. We added these tags using `WithTags` method during the initialization step of context

Reply:
ok perfect. missed that